### PR TITLE
fix: removes scrolling caused by overflow

### PR DESCRIPTION
**Pull request by rajatgupta24**
This PR fixes:
- removes scrolling caused by the overflow of the table component
        - adds `overflow-hidden` to stop vertical scroll
        - changes the width to `100%` instead of `100vw`
        - increases the width of table
- the num(in `admin/index`) is a number, passing a string gives an error

Signed-off-by: rajatgupta24 <rajat2411gupta@gmail.com>

### CODE REVIEW

File: web/src/lib/Components/Table.svelte [hunk 1]
```diff
@@ -13,4 +13,4 @@
   })
 </script>
 
-<div id={tableId} class="h-3/4 w-3/4 ag-theme-alpine-dark" />
+<div id={tableId} class="h-3/4 w-9/10 ag-theme-alpine-dark" />
```
Comment (ankitbrgy123) [inline web/src/lib/Components/Table.svelte:16]: Why this change? Does it not scale well with 3/4?

Comment (ankitm123) [inline web/src/lib/Components/Table.svelte:16]: Why this change?

Comment (ankitm123) [reply web/src/lib/Components/Table.svelte:16]: In the future, we could look at tools like percy which will show an image diff ...

File: web/src/routes/admin/pipelines/index.svelte [hunk 2]
```diff
@@ -78,11 +78,11 @@
...
 </svelte:head>
 
-<main class="h-full overflow-y-auto">
-  <div class="container px-6 mx-auto grid">
+<main class="h-full overflow-hidden">
+  <div class="container px-6 mx-auto grid overflow-hidden">
     <h2 class="my-6 text-2xl font-semibold text-gray-700 dark:text-gray-200">Pipelines</h2>
     <!-- New Table -->
-    <div class="w-screen h-screen overflow-hidden rounded-lg shadow-xs">
+    <div class="w-full h-screen overflow-hidden rounded-lg shadow-xs">
```
Comment (ankitm123) [inline web/src/routes/admin/pipelines/index.svelte:85]: Going from 100vw to 100%, interesting. Looks better?

File: web/src/lib/Components/Table.svelte (hunk 1 above)
Comment (rajatgupta24) [reply web/src/lib/Components/Table.svelte:16]: Yes, previously (at 75%), after fixing the overflow issue the table was not filling the space entirely, at 90% it utilizes the space and leaves an equal margin on both sides. 

At 75%
image

At 90%
image

File: web/src/routes/admin/pipelines/index.svelte (hunk 2 above)
Comment (rajatgupta24) [reply web/src/routes/admin/pipelines/index.svelte:85]: 100vw = the actual screen size
100% = the full width of the current component

```mermaid
  graph TD;
      A100vw-->F(B50vw);
      A100vw-->S(C50vw);
      F(B50vw)-->F1(C25vw);
      F(B50vw)-->S1(D25vw);
```

Now, for C and D 100% = 25vw

**Review by ankitm123 — APPROVED** [review]
